src/store/modules/GlobalStore.js

```diff
@@ -28,6 +35,15 @@ const GlobalStore = {
           commit('setHostName', hostName);
         })
         .catch(error => console.log(error));
+    },
+    getBmcTime({ commit }) {
+      api
+        .get('/xyz/openbmc_project/time/bmc')
+        .then(response => {
+          const bmcTime = response.data.data.Elapsed / 1000;
```

Comment:
when we upstream, should add a comment on why we are doing this 
// bmcTime is stored in microseconds, convert to millseconds

Reply:
@gtmills Is there ever a time that there wouldn't be a value sent in the `Elapsed` property?

Reply:
As long as that interface is implemented at this path, will have that property. 
https://github.com/openbmc/phosphor-dbus-interfaces/blob/master/xyz/openbmc_project/Time/EpochTime.interface.yaml#L4
I am fine assuming we do.